ctlearn/default_models/single_cnn.py

```diff
@@ -19,8 +20,10 @@ def single_cnn_model(data, model_params):
                 model.trainable = trainable_backbone
     else:
         sys.path.append(model_params["model_directory"])
-        engine_module = importlib.import_module(model_params["engine"]["module"])
-        engine = getattr(engine_module, model_params["engine"]["function"])
+        engine_module = importlib.import_module(model_params["engine_cnn"]["module"])
```

Comment:
Unify the naming convention. Here I would propose engine_cnn_module = ...

Reply:
okay I will update these names